ckanext/datagovuk/upload.py

```diff
@@ -0,0 +1,149 @@
+'''
+upload.py
+
+Contains functions that upload the resources/zipfiles to S3.
```

Comment:
Comment can be updated to just organograms as I don't think  we upload any other resources or zipfiles to s3.